caravel/viz.py

```diff
@@ -177,7 +177,7 @@ def get_df(self, query_obj=None):
                 if self.datasource.offset:
                     df.timestamp += timedelta(hours=self.datasource.offset)
         df.replace([np.inf, -np.inf], np.nan)
-        df = df.fillna(0)
+        df = df.fillna('NULL')
```

Comment:
It seems like this main create some problems where there's missing data expected to be numeric. Ultimately we want javascript's reserved `null` in the json. Would `df.fillna(None)` work?